web/app/js/components/StatusTable.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import * as d3 from 'd3';
 import { Link } from 'react-router-dom';
-import { Table, Tabs } from 'antd';
+import { Table, Tabs, Tooltip } from 'antd';
 import { toClassName, metricToFormatter } from './util/Utils.js';
 
 const getStatusDotCn = status => {
   return `status-dot-${status === "good" ? "green" : "grey"}`;
 }
 
+const statusDotExplanation = {
+  good: "has been added to the mesh",
+  neutral: "has not been added to the mesh"
+};
+
+const StatusDot = ({status}) => (
+  <Tooltip
+    placement="top"
+    title={`${status.name} ${statusDotExplanation[status.value]}`}
+  >
+    <div
+      className={`status-dot status-dot-${status.value}`}
+      key={status.name}
+    >&nbsp;</div>
+  </Tooltip>
```

Comment:
This is nit picky, but the wrapping of the text in the tooltip is a bit unfortunate in my demo setup:

![screen shot 2017-12-08 at 1 49 57 pm](https://user-images.githubusercontent.com/9226/33786685-d920586a-dc1e-11e7-8fff-8ed4d574f234.png)

Maybe we could split the message onto two lines, with the first line being the pod name, and the second line being a short message that won't wrap. Might also be worth asking @sterlingwhite to see if he has any suggestions.

Reply:
hmm yeahh good idea, I'll try putting the pod name on its own line and we'll see how that goes!